src/anyp/TrafficMode.h

```diff
@@ -135,30 +145,27 @@ class TrafficMode
     TrafficModeFlags flags_;
 };
 
-inline std::ostream &
+inline void
 TrafficMode::print(std::ostream &os) const
 {
     if (flags_.natIntercept)
-        os << " NAT intercepted";
+        os << "intercept ";
```

Comment:
AFAICT, if my other suggestions are accepted, then we should keep the leading spaces in this method instead of adding trailing spaces.

Reply:
Thank you.

FWIW, in a debugs() variable reporting/dumping context, it is often easier to deal with the extra leading space (by adding a variable or context description) than a trailing space.